va-gov/pages/burials-memorials/index.md

```diff
@@ -5,7 +5,7 @@ title: VA Burial Benefits and Memorial Items
 display_title: Burials and Memorials
 lastupdate:
 order: 1
-icon: star
+icon: icon-large white fa fa-star hub-background-burials
```

Comment:
I just moved the entire icon class here. I first moved the `fa-` stuff, then had to apply the hub's background since we can no longer generate that from the FA class, and at that point just decided to move it all.